static/app/utils/profiling/canvasScheduler.tsx

```diff
@@ -129,7 +129,7 @@ export class CanvasPoolManager {
     args: ArgumentTypes<FlamegraphEvents[K]>
   ): void {
     for (const scheduler of this.schedulers) {
-      scheduler.dispatch(event, args);
+      scheduler.dispatch(event, ...args);
```

Comment:
Is this correct? `scheduler.dispatch` only takes 2 arguments, spreading it like this means if additional args are passed, they'll be lost here.

Reply:
Nice catch. I actually want to collect rest args on L129 so that we can support variadic args here